modules/features/sharing/src/main/kotlin/au/com/shiftyjelly/pocketcasts/sharing/ui/SharePage.kt

```diff
@@ -71,14 +78,41 @@ internal fun VerticalSharePage(
     }
     Box(
         contentAlignment = Alignment.Center,
-        content = middleContent,
+        content = {
+            val pagerState = rememberPagerState(pageCount = { CardType.entries.size })
+            Column(
+                horizontalAlignment = Alignment.CenterHorizontally,
+            ) {
+                BoxWithConstraints(
+                    modifier = Modifier.weight(0.92f),
+                ) {
+                    val expectedVerticalCardPadding = remember(maxWidth, maxHeight) { (maxWidth - maxHeight / 1.5f) / 2 }
+                    HorizontalPager(
```

Comment:
I tested it on different layout sizes and it works well. 

I was about to suggest reusing an existing component `HorizontalPagerWrapper` but
realized that it takes the height of the tallest page for other pages which is not what we want here. 

No action needed.